src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.Arrays;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    private Truck truck = new Truck();
+    private Truck truck1 = new Truck();
+    private Truck truck2 = new Truck();
+
+    public List<Truck> get() {
```

Comment:
Don't forget about access modifiers for your custom fields in `Truck` class. You should never want to expose the object fields directly. They should be accessed through special methods (getters and setters).